tests/integration/components/ember-ace-test.js

```diff
@@ -167,11 +167,13 @@ module('Integration | Component | ember ace', function(hooks) {
     const { autocomplete } = this.component;
 
     await autocomplete.trigger();
-    assert.deepEqual(autocomplete.suggestions().mapBy('caption'), ['lhs', 'lhs2']);
-    assert.deepEqual(autocomplete.suggestions().mapBy('meta'), ['rhs', 'rhs2']);
+    assert.deepEqual(autocomplete.suggestions().mapBy('caption'), ['lhs2', 'lhs']); // in opposite order now, ok?
```

Comment:
When we were running this locally, it looked like ACE's autocomplete might be putting them in the opposite order and using `top` to visually change the order. Just one more way ACE is anti-accessibility.